projects/storefrontstyles/scss/layout/page-templates/_order-detail-page.scss

```diff
@@ -89,16 +93,11 @@
     }
   }
 
+  // TODO: clean it in 5.0, since it is not used
   cx-order cx-order-details-totals {
     margin-bottom: 32px;
   }
 
-  cx-tab-paragraph-container {
-    .active {
-      background-color: var(--cx-color-inverse);
-    }
-  }
-
```

Comment:
Why do we remove this part?

Reply:
It is moved to tab-paragraph-container.scss